Unable to import library

I tried installing this library following the instructions for both regular and lazy loaded modules, however I'm getting this error a bunch of times: 
```
node_modules/@angular/core/core"' has no exported member 'ɵɵFactoryDeclaration'.

78     static ɵfac: i0.ɵɵFactoryDeclaration<TourService<any>, never>;
```

Note that I have required versions of dependecies:
```
"@angular/animations": "9.1.13",
"@angular/cdk": "9.2.4",
"@angular/common": "9.1.13",
"@angular/core": "9.1.13",
"@angular/material": "9.2.4",
"typescript": "3.8.3",
"ngx-ui-tour-core": "^7.0.1",
"ngx-ui-tour-md-menu": "^7.0.1",
```

I wonder if I cannot use latest version if I'm not using Angular 12, the description says supports Angular9+ but does it even in the latest version or just before support for Angular 10 is added?

**Edit:** Using 7+ will result in these errors, 6.1.0 seems fine.

Any help will be appreciated.

> Are you using Ivy compiler?
> https://stackoverflow.com/a/58522911

Hello there friends !
I have kinda the same problem while I'm working with Angular 10.
PS : I have Ivy Compiler activated in my tsconfig

`ERROR in node_modules/ngx-ui-tour-core/lib/tour-backdrop.service.d.ts:21:21 - error TS2694: Namespace '".../node_modules/@angular/core/core"' has no exported member 'ɵɵ
FactoryDeclaration'.

21     static ɵfac: i0.ɵɵFactoryDeclaration<TourBackdropService, never>;
                       ~~~~~~~~~~~~~~~~~~~~
node_modules/ngx-ui-tour-core/lib/tour-backdrop.service.d.ts:22:22 - error TS2694: Namespace '".../node_modules/@angular/core/core"' has no exported member 'ɵɵInjectabl
eDeclaration'.

22     static ɵprov: i0.ɵɵInjectableDeclaration<TourBackdropService>;
                        ~~~~~~~~~~~~~~~~~~~~~~~
node_modules/ngx-ui-tour-core/lib/tour-hotkey-listener.component.d.ts:12:21 - error TS2694: Namespace '".../node_modules/@angular/core/core"' has no exported member 'ɵɵ
FactoryDeclaration'.

12     static ɵfac: i0.ɵɵFactoryDeclaration<TourHotkeyListenerComponent, never>;
                       ~~~~~~~~~~~~~~~~~~~~
node_modules/ngx-ui-tour-core/lib/tour-hotkey-listener.component.d.ts:13:21 - error TS2694: Namespace '".../node_modules/@angular/core/core"' has no exported member 'ɵɵ
ComponentDeclaration'.

13     static ɵcmp: i0.ɵɵComponentDeclaration<TourHotkeyListenerComponent, "tour-hotkey-listener", never, {}, {}, never, ["*"]>;
                       ~~~~~~~~~~~~~~~~~~~~~~
node_modules/ngx-ui-tour-core/lib/tour.module.d.ts:10:21 - error TS2694: Namespace '".../node_modules/@angular/core/core"' has no exported member 'ɵɵFactoryDeclaration'
.

10     static ɵfac: i0.ɵɵFactoryDeclaration<TourModule, never>;
                       ~~~~~~~~~~~~~~~~~~~~
node_modules/ngx-ui-tour-core/lib/tour.module.d.ts:11:21 - error TS2694: Namespace '".../node_modules/@angular/core/core"' has no exported member 'ɵɵNgModuleDeclaration
'.

11     static ɵmod: i0.ɵɵNgModuleDeclaration<TourModule, [typeof i1.TourHotkeyListenerComponent], [typeof i2.CommonModule, typeof i3.RouterModule], [typeof i1.TourHotkeyListenerComponent]>;
                       ~~~~~~~~~~~~~~~~~~~~~
node_modules/ngx-ui-tour-core/lib/tour.module.d.ts:12:21 - error TS2694: Namespace '".../node_modules/@angular/core/core"' has no exported member 'ɵɵInjectorDeclaration
'.

12     static ɵinj: i0.ɵɵInjectorDeclaration<TourModule>;
                       ~~~~~~~~~~~~~~~~~~~~~
node_modules/ngx-ui-tour-core/lib/tour.service.d.ts:78:21 - error TS2694: Namespace '".../node_modules/@angular/core/core"' has no exported member 'ɵɵFactoryDeclaration
'.

78     static ɵfac: i0.ɵɵFactoryDeclaration<TourService<any>, never>;
                       ~~~~~~~~~~~~~~~~~~~~
node_modules/ngx-ui-tour-core/lib/tour.service.d.ts:79:22 - error TS2694: Namespace '".../node_modules/@angular/core/core"' has no exported member 'ɵɵInjectableDeclarat
ion'.

79     static ɵprov: i0.ɵɵInjectableDeclaration<TourService<any>>;
                        ~~~~~~~~~~~~~~~~~~~~~~~
node_modules/ngx-ui-tour-ng-bootstrap/lib/ng-bootstrap-tour.service.d.ts:5:21 - error TS2694: Namespace '".../node_modules/@angular/core/core"' has no exported member '
ɵɵFactoryDeclaration'.

5     static ɵfac: i0.ɵɵFactoryDeclaration<NgbTourService, never>;
                      ~~~~~~~~~~~~~~~~~~~~
node_modules/ngx-ui-tour-ng-bootstrap/lib/ng-bootstrap-tour.service.d.ts:6:22 - error TS2694: Namespace '".../node_modules/@angular/core/core"' has no exported member '
ɵɵInjectableDeclaration'.

6     static ɵprov: i0.ɵɵInjectableDeclaration<NgbTourService>;
                       ~~~~~~~~~~~~~~~~~~~~~~~
node_modules/ngx-ui-tour-ng-bootstrap/lib/ng-bootstrap.module.d.ts:14:21 - error TS2694: Namespace '".../node_modules/@angular/core/core"' has no exported member 'ɵɵFac
toryDeclaration'.

14     static ɵfac: i0.ɵɵFactoryDeclaration<TourNgBootstrapModule, never>;
                       ~~~~~~~~~~~~~~~~~~~~
node_modules/ngx-ui-tour-ng-bootstrap/lib/ng-bootstrap.module.d.ts:15:21 - error TS2694: Namespace '".../node_modules/@angular/core/core"' has no exported member 'ɵɵNgM
oduleDeclaration'.

15     static ɵmod: i0.ɵɵNgModuleDeclaration<TourNgBootstrapModule, [typeof i1.TourAnchorNgBootstrapDirective, typeof i1.TourAnchorNgBootstrapPopoverDirective, typeof i2.TourStepTemplateComponent], [typeof i3.Comm
onModule, typeof i4.NgbPopoverModule, typeof i5.TourModule], [typeof i1.TourAnchorNgBootstrapDirective, typeof i1.TourAnchorNgBootstrapPopoverDirective, typeof i2.TourStepTemplateComponent]>;
                       ~~~~~~~~~~~~~~~~~~~~~
node_modules/ngx-ui-tour-ng-bootstrap/lib/ng-bootstrap.module.d.ts:16:21 - error TS2694: Namespace '".../node_modules/@angular/core/core"' has no exported member 'ɵɵInj
ectorDeclaration'.

16     static ɵinj: i0.ɵɵInjectorDeclaration<TourNgBootstrapModule>;
                       ~~~~~~~~~~~~~~~~~~~~~
node_modules/ngx-ui-tour-ng-bootstrap/lib/tour-anchor.directive.d.ts:10:21 - error TS2694: Namespace '".../node_modules/@angular/core/core"' has no exported member 'ɵɵF
actoryDeclaration'.

10     static ɵfac: i0.ɵɵFactoryDeclaration<TourAnchorNgBootstrapPopoverDirective, never>;
                       ~~~~~~~~~~~~~~~~~~~~
node_modules/ngx-ui-tour-ng-bootstrap/lib/tour-anchor.directive.d.ts:11:21 - error TS2694: Namespace '".../node_modules/@angular/core/core"' has no exported member 'ɵɵD
irectiveDeclaration'.

11     static ɵdir: i0.ɵɵDirectiveDeclaration<TourAnchorNgBootstrapPopoverDirective, "[tourAnchor]", never, {}, {}, never>;
                       ~~~~~~~~~~~~~~~~~~~~~~
node_modules/ngx-ui-tour-ng-bootstrap/lib/tour-anchor.directive.d.ts:26:21 - error TS2694: Namespace '".../node_modules/@angular/core/core"' has no exported member 'ɵɵF
actoryDeclaration'.

26     static ɵfac: i0.ɵɵFactoryDeclaration<TourAnchorNgBootstrapDirective, [null, null, null, { host: true; }, null]>;
                       ~~~~~~~~~~~~~~~~~~~~
node_modules/ngx-ui-tour-ng-bootstrap/lib/tour-anchor.directive.d.ts:27:21 - error TS2694: Namespace '".../node_modules/@angular/core/core"' has no exported member 'ɵɵD
irectiveDeclaration'.

27     static ɵdir: i0.ɵɵDirectiveDeclaration<TourAnchorNgBootstrapDirective, "[tourAnchor]", never, { "tourAnchor": "tourAnchor"; }, {}, never>;
                       ~~~~~~~~~~~~~~~~~~~~~~
node_modules/ngx-ui-tour-ng-bootstrap/lib/tour-step-template.component.d.ts:18:21 - error TS2694: Namespace '".../node_modules/@angular/core/core"' has no exported memb
er 'ɵɵFactoryDeclaration'.

18     static ɵfac: i0.ɵɵFactoryDeclaration<TourStepTemplateComponent, never>;
                       ~~~~~~~~~~~~~~~~~~~~
node_modules/ngx-ui-tour-ng-bootstrap/lib/tour-step-template.component.d.ts:19:21 - error TS2694: Namespace '".../node_modules/@angular/core/core"' has no exported memb
er 'ɵɵComponentDeclaration'.

19     static ɵcmp: i0.ɵɵComponentDeclaration<TourStepTemplateComponent, "tour-step-template", never, { "stepTemplate": "stepTemplate"; }, {}, ["stepTemplateContent"], never>;
                       ~~~~~~~~~~~~~~~~~~~~~~
node_modules/ngx-ui-tour-ng-bootstrap/lib/tour-step-template.service.d.ts:7:21 - error TS2694: Namespace '".../node_modules/@angular/core/core"' has no exported member
'ɵɵFactoryDeclaration'.

7     static ɵfac: i0.ɵɵFactoryDeclaration<TourStepTemplateService, never>;
                      ~~~~~~~~~~~~~~~~~~~~
node_modules/ngx-ui-tour-ng-bootstrap/lib/tour-step-template.service.d.ts:8:22 - error TS2694: Namespace '".../node_modules/@angular/core/core"' has no exported member
'ɵɵInjectableDeclaration'.

8     static ɵprov: i0.ɵɵInjectableDeclaration<TourStepTemplateService>;
`

--- a/projects/ngx-ui-tour-console/CHANGELOG.md
+++ b/projects/ngx-ui-tour-console/CHANGELOG.md
@@ -1,3 +1,11 @@
+<a name="7.0.2"></a>
+
+# [7.0.2](https://github.com/hakimio/ngx-ui-tour) (2021-09-23)
+
+### Fixes
+
+- Switched back to "View Engine" compilation mode to fix Angular v11 and older builds
+
 <a name="7.0.1"></a>
 
 # [7.0.1](https://github.com/hakimio/ngx-ui-tour) (2021-09-09)

--- a/projects/ngx-ui-tour-console/package.json
+++ b/projects/ngx-ui-tour-console/package.json
@@ -1,6 +1,6 @@
 {
   "name": "ngx-ui-tour-console",
-  "version": "7.0.1",
+  "version": "7.0.2",
   "repository": "https://github.com/hakimio/ngx-ui-tour.git",
   "author": "Isaac Mann <isaacplmann@gmail.com>",
   "license": "MIT",
@@ -17,7 +17,7 @@
   "peerDependencies": {
     "@angular/common": ">=9.0.0 <13.0.0",
     "@angular/core": ">=9.0.0 <13.0.0",
-    "ngx-ui-tour-core": "7.0.1",
+    "ngx-ui-tour-core": "7.0.2",
     "typescript": ">=3.8.0"
   },
   "dependencies": {

--- a/projects/ngx-ui-tour-console/tsconfig.lib.prod.json
+++ b/projects/ngx-ui-tour-console/tsconfig.lib.prod.json
@@ -4,6 +4,6 @@
     "declarationMap": false
   },
   "angularCompilerOptions": {
-    "compilationMode": "partial"
+    "enableIvy": false
   }
 }

--- a/projects/ngx-ui-tour-core/CHANGELOG.md
+++ b/projects/ngx-ui-tour-core/CHANGELOG.md
@@ -1,3 +1,11 @@
+<a name="7.0.2"></a>
+
+# [7.0.2](https://github.com/hakimio/ngx-ui-tour) (2021-09-23)
+
+### Fixes
+
+- Switched back to "View Engine" compilation mode to fix Angular v11 and older builds
+
 <a name="7.0.1"></a>
 
 # [7.0.1](https://github.com/hakimio/ngx-ui-tour) (2021-09-09)

--- a/projects/ngx-ui-tour-core/package.json
+++ b/projects/ngx-ui-tour-core/package.json
@@ -1,6 +1,6 @@
 {
   "name": "ngx-ui-tour-core",
-  "version": "7.0.1",
+  "version": "7.0.2",
   "description": "UI tour library for Angular 9+",
   "keywords": [
     "angular",

--- a/projects/ngx-ui-tour-core/tsconfig.lib.prod.json
+++ b/projects/ngx-ui-tour-core/tsconfig.lib.prod.json
@@ -4,6 +4,6 @@
     "declarationMap": false
   },
   "angularCompilerOptions": {
-    "compilationMode": "partial"
+    "enableIvy": false
   }
 }

--- a/projects/ngx-ui-tour-md-menu/CHANGELOG.md
+++ b/projects/ngx-ui-tour-md-menu/CHANGELOG.md
@@ -1,3 +1,11 @@
+<a name="7.0.2"></a>
+
+# [7.0.2](https://github.com/hakimio/ngx-ui-tour) (2021-09-23)
+
+### Fixes
+
+- Switched back to "View Engine" compilation mode to fix Angular v11 and older builds
+
 <a name="7.0.1"></a>
 
 # [7.0.1](https://github.com/hakimio/ngx-ui-tour) (2021-09-09)

--- a/projects/ngx-ui-tour-md-menu/package.json
+++ b/projects/ngx-ui-tour-md-menu/package.json
@@ -1,6 +1,6 @@
 {
   "name": "ngx-ui-tour-md-menu",
-  "version": "7.0.1",
+  "version": "7.0.2",
   "description": "UI tour library for Angular 9+",
   "keywords": [
     "angular",
@@ -26,7 +26,7 @@
     "@angular/animations": ">=9.0.0 <13.0.0",
     "@angular/cdk": ">=9.0.0 <13.0.0",
     "@angular/material": ">=9.0.0 <13.0.0",
-    "ngx-ui-tour-core": "7.0.1",
+    "ngx-ui-tour-core": "7.0.2",
     "typescript": ">=3.8.0"
   }
 }

--- a/projects/ngx-ui-tour-md-menu/tsconfig.lib.prod.json
+++ b/projects/ngx-ui-tour-md-menu/tsconfig.lib.prod.json
@@ -4,6 +4,6 @@
     "declarationMap": false
   },
   "angularCompilerOptions": {
-    "compilationMode": "partial"
+    "enableIvy": false
   }
 }

--- a/projects/ngx-ui-tour-ng-bootstrap/CHANGELOG.md
+++ b/projects/ngx-ui-tour-ng-bootstrap/CHANGELOG.md
@@ -1,3 +1,11 @@
+<a name="9.0.2"></a>
+
+# [9.0.2](https://github.com/hakimio/ngx-ui-tour) (2021-09-23)
+
+### Fixes
+
+- Switched back to "View Engine" compilation mode to fix Angular v11 and older builds
+
 <a name="9.0.1"></a>
 
 # [9.0.1](https://github.com/hakimio/ngx-ui-tour) (2021-09-09)

--- a/projects/ngx-ui-tour-ng-bootstrap/package.json
+++ b/projects/ngx-ui-tour-ng-bootstrap/package.json
@@ -1,6 +1,6 @@
 {
   "name": "ngx-ui-tour-ng-bootstrap",
-  "version": "9.0.1",
+  "version": "9.0.2",
   "description": "UI tour library for Angular 9+",
   "keywords": [
     "angular",
@@ -20,7 +20,7 @@
     "@angular/common": ">=9.0.0 <13.0.0",
     "@angular/core": ">=9.0.0 <13.0.0",
     "@ng-bootstrap/ng-bootstrap": ">=7.0.0",
-    "ngx-ui-tour-core": "7.0.1",
+    "ngx-ui-tour-core": "7.0.2",
     "typescript": ">=3.8.0"
   },
   "dependencies": {

--- a/projects/ngx-ui-tour-ng-bootstrap/tsconfig.lib.prod.json
+++ b/projects/ngx-ui-tour-ng-bootstrap/tsconfig.lib.prod.json
@@ -4,6 +4,6 @@
     "declarationMap": false
   },
   "angularCompilerOptions": {
-    "compilationMode": "partial"
+    "enableIvy": false
   }
 }

--- a/projects/ngx-ui-tour-ngx-bootstrap/CHANGELOG.md
+++ b/projects/ngx-ui-tour-ngx-bootstrap/CHANGELOG.md
@@ -1,3 +1,11 @@
+<a name="7.0.2"></a>
+
+# [7.0.2](https://github.com/hakimio/ngx-ui-tour) (2021-09-23)
+
+### Fixes
+
+- Switched back to "View Engine" compilation mode to fix Angular v11 and older builds
+
 <a name="7.0.1"></a>
 
 # [7.0.1](https://github.com/hakimio/ngx-ui-tour) (2021-09-09)

--- a/projects/ngx-ui-tour-ngx-bootstrap/package.json
+++ b/projects/ngx-ui-tour-ngx-bootstrap/package.json
@@ -1,6 +1,6 @@
 {
   "name": "ngx-ui-tour-ngx-bootstrap",
-  "version": "7.0.1",
+  "version": "7.0.2",
   "description": "UI tour library for Angular 9+",
   "keywords": [
     "angular",
@@ -20,7 +20,7 @@
     "@angular/common": ">=9.0.0 <13.0.0",
     "@angular/core": ">=9.0.0 <13.0.0",
     "ngx-bootstrap": ">=3.1.0 <8.0.0",
-    "ngx-ui-tour-core": "7.0.1",
+    "ngx-ui-tour-core": "7.0.2",
     "typescript": ">=3.8.0"
   },
   "dependencies": {

--- a/projects/ngx-ui-tour-ngx-bootstrap/tsconfig.lib.prod.json
+++ b/projects/ngx-ui-tour-ngx-bootstrap/tsconfig.lib.prod.json
@@ -4,6 +4,6 @@
     "declarationMap": false
   },
   "angularCompilerOptions": {
-    "compilationMode": "partial"
+    "enableIvy": false
   }
 }

--- a/projects/ngx-ui-tour-tui-dropdown/CHANGELOG.md
+++ b/projects/ngx-ui-tour-tui-dropdown/CHANGELOG.md
@@ -1,3 +1,11 @@
+<a name="1.0.1"></a>
+
+# [1.0.1](https://github.com/hakimio/ngx-ui-tour) (2021-09-23)
+
+### Fixes
+
+- Switched back to "View Engine" compilation mode to fix Angular v11 and older builds
+
 <a name="1.0.0"></a>
 
 # [1.0.0](https://github.com/hakimio/ngx-ui-tour) (2021-09-20)

--- a/projects/ngx-ui-tour-tui-dropdown/package.json
+++ b/projects/ngx-ui-tour-tui-dropdown/package.json
@@ -14,14 +14,14 @@
   "repository": "https://github.com/hakimio/ngx-ui-tour.git",
   "author": "Tomas Rimkus <tomas.r@programmer.net>",
   "license": "MIT",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "peerDependencies": {
     "@angular/common": ">=9.0.0 <13.0.0",
     "@angular/core": ">=9.0.0 <13.0.0",
     "@taiga-ui/core": "^2.15.0",
     "@taiga-ui/cdk": "^2.15.0",
     "@taiga-ui/icons": "^2.15.0",
-    "ngx-ui-tour-core": "7.0.1",
+    "ngx-ui-tour-core": "7.0.2",
     "typescript": ">=3.8.0"
   },
   "dependencies": {

--- a/projects/ngx-ui-tour-tui-dropdown/tsconfig.lib.prod.json
+++ b/projects/ngx-ui-tour-tui-dropdown/tsconfig.lib.prod.json
@@ -5,6 +5,6 @@
     "declarationMap": false
   },
   "angularCompilerOptions": {
-    "compilationMode": "partial"
+    "enableIvy": false
   }
 }

--- a/projects/ngx-ui-tour-tui-hint/CHANGELOG.md
+++ b/projects/ngx-ui-tour-tui-hint/CHANGELOG.md
@@ -1,3 +1,11 @@
+<a name="1.0.2"></a>
+
+# [1.0.2](https://github.com/hakimio/ngx-ui-tour) (2021-09-23)
+
+### Fixes
+
+- Switched back to "View Engine" compilation mode to fix Angular v11 and older builds
+
 <a name="1.0.1"></a>
 
 # [1.0.1](https://github.com/hakimio/ngx-ui-tour) (2021-09-23)

--- a/projects/ngx-ui-tour-tui-hint/package.json
+++ b/projects/ngx-ui-tour-tui-hint/package.json
@@ -14,14 +14,14 @@
   "repository": "https://github.com/hakimio/ngx-ui-tour.git",
   "author": "Tomas Rimkus <tomas.r@programmer.net>",
   "license": "MIT",
-  "version": "1.0.1",
+  "version": "1.0.2",
   "peerDependencies": {
     "@angular/common": ">=9.0.0 <13.0.0",
     "@angular/core": ">=9.0.0 <13.0.0",
     "@taiga-ui/core": "^2.15.0",
     "@taiga-ui/cdk": "^2.15.0",
     "@taiga-ui/icons": "^2.15.0",
-    "ngx-ui-tour-core": "7.0.1",
+    "ngx-ui-tour-core": "7.0.2",
     "typescript": ">=3.8.0"
   },
   "dependencies": {

--- a/projects/ngx-ui-tour-tui-hint/tsconfig.lib.prod.json
+++ b/projects/ngx-ui-tour-tui-hint/tsconfig.lib.prod.json
@@ -5,6 +5,6 @@
     "declarationMap": false
   },
   "angularCompilerOptions": {
-    "compilationMode": "partial"
+    "enableIvy": false
   }
 }

